pkg/controller/replication_controller.go

```diff
@@ -64,6 +64,10 @@ const (
 	// 3000 pods. Just creation is limited to 30qps, and watching happens with
 	// ~10-30s latency/pod at scale.
 	ExpectationsTimeout = 6 * time.Minute
```

Comment:
Yeah, was going to send a follow up pr, and observe latencies in production for a bit. That timeout isn't *that* critical unless we drop events, would you prefer I did so in this one?